torch/distributed/nn/jit/templates/remote_module.py.template

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+from typing import Tuple, Any
+
+import torch
+import torch.distributed.rpc as rpc
+from torch import Tensor, nn
+from torch._jit_internal import Future, RRef
+from ${module_interface_cls_module_name} import ${module_interface_cls_name} as module_interface_cls  # noqa
+
+
+class _RemoteModule(nn.Module):  # noqa
+    module_rref: RRef[module_interface_cls]  # noqa
+    is_scriptable: bool
+    global_unique_name: str
+
+    def __init__(self, module_rref, is_scriptable, global_unique_name):
+        super().__init__()
+        self.module_rref = module_rref
+        self.is_scriptable = is_scriptable
+        self.global_unique_name = global_unique_name
```

Comment:
We can get rid of global_unique_name since it is no longer used?

Reply:
I want to make RemoteModule a convenient wrapper.

This field will be used by post processing. For example, checkpoint and re-sharding will need to locate the shard wrapped by RemoteModule.